.mockery.yaml

```
@@ -0,0 +1,23 @@
+      ReprovideStore: {}
+      MetadataStore: {}
+      BlockDownloader: {}
+      IPFSPinService: {}
+      UploadService: {}
+      BlockService: {}
+  go.lumeweb.com/portal-plugin-ipfs/internal/protocol/store:
+    config:
+      dir: ./internal/testing/mocks
+      pkgname: "mocks"
+    interfaces:
+      MockBlockstore: {}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)